mod/05_standard/set/type/uri.rb

```diff
@@ -0,0 +1,11 @@
+format do
+  include Phrase::Format
+
+  view :core do |args|
+    build_link _render_raw(args), render_title(args) || card.name
+  end
+
+  view :uri do |args|
+    build_link _render_raw(args)
```

Comment:
aliasing uri and url is a minor point.  we already have url and want to make it work; no big deal adding uri, imo, but I don't really care about that.

I do care strongly about NOT turning the nest (inclusion) syntax into a programming language (which would dramatically complicate building the nest interface, which is the centerpiece of our strategy for making wagneering approachable).  So I don't like the _raw idea.  (especially since underscore already has a meaning in name contexts and it doesn't have to deal with views).

Basically, I think we just need a uri_link view.  Isn't that easier?